### PR TITLE
Docs

### PR DESCRIPTION
**Pull request by aburrell**
# Description

This adds an up-to-date documentation for the module.

# Type of change

Please delete options that are not relevant.

- New feature (non-breaking change which adds functionality)
- This change requires a documentation update

# How Has This Been Tested?

Make local changes outlined here: https://github.com/CrossNox/m2r2/pull/14

Then run `make html` and check out the files in the .build/html directory

## Test Configuration
* Operating system: OS X Mojave
* Version number: Python 3.7
* Any details about your local setup that are relevant: custom updated version of m2r2

# Checklist:

- [x] Make sure you are merging into the ``develop`` (not ``master``) branch
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules
- [x] Add a note to ``CHANGELOG.md``, summarizing the changes

If this is a release PR, replace the first item of the above checklist with the
release checklist on the pysat wiki:
https://github.com/pysat/pysat/wiki/Checklist-for-Release

### CODE REVIEW

**Comment by aburrell** [comment]
Version 0.2.6 of m2r2 now works and readthedocs reflects this.  We're good to go.

File: docs/develop_guide/contributing.rst [hunk 1]
```diff
@@ -0,0 +1 @@
+.. mdinclude:: ../../CONTRIBUTING.md
```
Comment (rstoneback) [inline docs/develop_guide/contributing.rst:1]: There isn't a contributing.md in the repo so this line causes a few errors when making the documentation.

**Review by rstoneback — COMMENTED** [review]
Looks good overall @aburrell, thanks! Found one issue, the docs link to a file not present in the repo (contributing.md).

**Comment by aburrell** [comment]
@rstoneback that document was erroneously introduced directly to the main branch.  So this will be solved when prepping the release candidate branch.

**Review by rstoneback — APPROVED** [review]

**Pull request merged**